removes static from lib names as it caused problems when linking
J. Sassmannshausen
diff --git a/hisat2-2.2.1.orig/Makefile b/hisat2-2.2.1/Makefile
index 60445ce..0b422f8 100644
--- a/hisat2-2.2.1.orig/Makefile
+++ b/hisat2-2.2.1/Makefile
@@ -94,7 +94,7 @@ SRA_LIB =
 SERACH_INC = 
 ifeq (1,$(USE_SRA))
 	SRA_DEF = -DUSE_SRA
-	SRA_LIB = -lncbi-ngs-c++-static -lngs-c++-static -lncbi-vdb-static -ldl
+	SRA_LIB = -lncbi-ngs-c++ -lngs-c++ -lncbi-vdb -ldl
 	SEARCH_INC += -I$(NCBI_NGS_DIR)/include -I$(NCBI_VDB_DIR)/include
 	SEARCH_LIBS += -L$(NCBI_NGS_DIR)/lib64 -L$(NCBI_VDB_DIR)/lib64
 endif
